collections/brandedUserProfile.js

```diff
@@ -314,30 +349,38 @@ BrandedUserSchema = new SimpleSchema({
     improvement: fields.long_string(o), // internal.orientation.improvement
     willYouAttend: fields.string(o), // internal.orientation.willYouAttend
     registered: fields.bool(o) // internal.registeredOrientation
-  }), optional: true},
+  }),
+  optional: true},
   _parking: {type: new SimpleSchema({
     bringCar: fields.bool(o) // _internal.needsParking
-  }), optional: true},
+  }),
+  optional: true},
   _phone: fields.string(o),
   _previousPhone: fields.string(o),
   _profile: {type: new SimpleSchema({
     description: fields.profile_description(o),
     parent: fields.bool(o),
     plannedApplication: fields.planned_application(o),
     setCanTextFalse: fields.bool(o)
-  }), optional: true},
+  }),
+  optional: true},
   _responseBlackBox: {
     type: Object,
     blackbox: true,
     optional: true
   }
-});
+})
 
 BrandedUserProfiles = new Mongo.Collection('brandedUserProfiles')
 BrandedUserProfiles.attachSchema(BrandedUserSchema)
 
+const DATED_FIELDS = {
+  '_contactSettings.canText': '_contactSettings.canTextLastModified',
+  'holoceneContext.currentContext': 'holoceneContext.lastMessageDate'
+}
+
 BrandedUserProfiles.before.update((userId, doc, fieldNames, modifier) => {
-  if (modifier.$set && modifier.$set['_contactSettings.canText'] !== undefined) {
-    modifier.$set['_contactSettings.canTextLastModified'] = new Date()
-  }
-});
```

Comment:
Did you mean to remove this snippet?  I suspect an incorrectly resolved merge conflict.

Reply:
he just moved it to the DATED_FIELDS const

Reply:
Indeed, I'm blind.  Thanks for clarifying!

Reply:
Yeah, I did. There is code which should achieve the same thing in the same place; did you see that?